### DISCUSSION TITLE
Aircraft Waypoints including Guard Mode

### DISCUSSION THREAD

**Discussion by ayylmaoRotE** [Phobos-developers/Phobos#873] [Ideas and feedback]
Request to implement the Tiberian Sun aircraft waypoint mode, where any aircraft is allowed to follow a waypoint path and take commands on it, such as guard area. In TS, once the aircraft ammo is depleted they go back and refill, then return to their waypoint path.

No particular tag I can think of.

**Comment by TaranDahl** [comment]
https://github.com/Phobos-developers/Phobos/pull/1654
